### PR TITLE
DID file Ftp server upload

### PR DESCRIPTION
**Pull request by mpetruzzellamalagon-ionos**
Good morning,

This is Mario from Arsys Spain, I belong to Sara Madariaga's team and I recently started working in Arsys self-descriptions files for the Gaia-X project.

First of all, let me thank you for sharing this tool, We've been working on this issue just for a few weeks and it's been really useful for us to understand how the signing process works and all the steps we have to follow to complete it.

I would like to share a little update we developed in our side for testing our files, maybe it may be helpful for other people who are getting into Gaia-X world and begin to use this tool.

In short, the generated DID file can be uploaded directly to the server (if configuration is enabled) so it's not neccesary for you to run the script twice to complete the process.

I hope we can keep in touch, and thanks a lot for your time.

Take care and best regards, 

## Proposed Changes

  - library ssh2-sftp-client
  - FTP config values in .env file
  - index.js will try to update the file if FTP_ENABLED is set to true

### CODE REVIEW

**Comment by kettenbach-it** [comment]
I wouldn't merge that. Following a clear separation of concerns, it's not the job of this program to do file transfers.
